main.js

```diff
@@ -22,15 +22,16 @@ function createWindow(on_load) {
     newWindow.loadURL('file://' + __dirname + '/index.html');
 }
 
-function newDocument() {
+function newProject() {
     createWindow(function(win) {
-        win.webContents.send('new-document');
+        win.webContents.send('new-project');
     });
 }
 
-function openDocument(path) {
+function openProject(path) {
     createWindow(function(win) {
-        win.webContents.send('open-document', path);
+        path = path.replace('app.tbinfo','');
```

Comment:
Shall I move the whole click function callback to one function to keep the view and functionality seperated? I think it would make the code a lot cleaner. But the choice is yours.